### PR TITLE
INFRA-1755 removed distributed testing code snippets

### PR DESCRIPTION
**Pull request by E1P3**

### CODE REVIEW

**Review by yarivus — DISMISSED** [review]
LGTM

File: build.gradle [hunk 1]
```diff
@@ -224,7 +224,6 @@ apply plugin: 'project-report'
 apply plugin: 'com.github.ben-manes.versions'
 apply plugin: 'net.corda.plugins.publish-utils'
 apply plugin: 'com.jfrog.artifactory'
-apply plugin: 'com.r3.testing.distributed-testing'
```
Comment (ronanbrowne) [inline build.gradle:228]: @E1P3 Search the file there is another entry for "distributed-testing" above where the plugin is declared in the Build scrip block , This is the bit which declares this plugin as a build dependency - we want to remove this also

**Review by ronanbrowne — CHANGES_REQUESTED** [review]

**Comment by ronanbrowne** [comment]
@E1P3 check PR gates, so there's further code referencing it through out project that needs removal , ensure you run a gradle command locally to test the projects build code compiles (even ./gradlew help in this case should catch these issues)

**Review by yarivus — DISMISSED** [review]
LGTM

File: build.gradle [hunk 2]
```diff
@@ -1,4 +1,3 @@
-import com.r3.testing.DistributeTestsBy
 import com.r3.testing.PodLogLevel
```
Comment (ronanbrowne) [inline build.gradle]: should be removed

**Review by ronanbrowne — CHANGES_REQUESTED** [review]

File: gradle.properties [hunk 3]
```diff
@@ -5,3 +5,4 @@ owasp.failOnError=false
 owasp.failBuildOnCVSS=11.0
 compilation.allWarningsAsErrors=false
 test.parallel=false
+docker_plugin_version=5.3.0
```
Comment (ronanbrowne) [inline gradle.properties]: @E1P3 Remove from here and put in constants. properties

then update build.gradle file to reference that in a similar manner to here https://github.com/corda/corda/blob/6d0875076f6e293fb503ba03b35e9b09b2b92862/build.gradle#L61

**Review by ronanbrowne — CHANGES_REQUESTED** [review]

**Review by yarivus — COMMENTED** [review]
LGTM

**Review by ronanbrowne — APPROVED** [review]
lgtm

**Review by yarivus — COMMENTED** [review]
@ronanbrowne - this is a leftover PR from @E1P3 
Can you please indicate the intent of the PR - R we proceeding to merge it ?

**Comment by ronanbrowne** [comment]
Closing will revisit this fresh as this PR is quite stale and I would like to verify some things here

**Pull request closed without merging**